src/commands/gaming/Wine/Stable.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/sh -e
+
+. ../../common-script.sh
+
+installDependencies() {
```

Comment:
```suggestion
enableMultiLib
```
This should get changed to "enableMultiLib